src/views/Trustees.tsx

```diff
@@ -56,7 +57,13 @@ const Trustees = () => {
             onContactSave={updateTrustee}
             onAddressSave={updateTrustee}
             onRemove={removeTrustee}
-            onCorrect={(value: any) => setComplete(value)}
+            onCorrect={(value: any) => {
```

Comment:
Probably here or somewhere else, we'll need to modify the value of **complete**

Reply:
The value of `complete` should not be changed by the `onCorrect` function. Marking a trustee as 'correct' by ticking the checkbox does not necessarily mean that the trustee information is 'complete'. The value of `complete` needs to be determined by different metric. There is a task in DevOps for me to make this change.

Reply:
I see, makes sense. That's why I say probably here or somewhere else, because you were using useState for the complete value, which means is expected to be modified at some point.